Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -38,6 +38,17 @@
       Cloth: 100
   - type: StaticPrice
     price: 5
+  - type: ItemSlots
```

Comment:
you need to add ContainerContainer component

test fail is real
![image](https://github.com/space-wizards/space-station-14/assets/96445749/da0459ee-0cd7-4374-9e85-a0c6bfa878e8)